src/js/xhr.js

```diff
@@ -28,8 +32,8 @@
  * @param  {Function}       callback  The callback function
  * @returns {Object}                  The request
  */
-vjs.xhr = function(options, callback){
-  var XHR, request, urlInfo, winLoc, fileUrl, crossOrigin, abortTimeout, successHandler, errorHandler;
+var xhr = function(options, callback){
```

Comment:
we should swap this file out for `xhr` from npm, but we can do that later, not even in the 5.0 but maybe in a 5.1 release, even.